spec/features/channel_select_multiple_contents_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.feature 'Multiple contents select', type: :feature do
+  let!(:organization) { create(:organization) }
+  let!(:video) { create(:video, organization: organization) }
+  let!(:gallery) { create(:gallery, organization: organization) }
+  let!(:admin) { create(:admin, organization: organization) }
+
+  CHANNEL_NAME = 'select multiple example channel'
+  it 'creates channel with multiple contents, then updates it' do
+    visit '/'
+    click_link 'Sign In'
+    fill_in 'Email', with: admin.email
+    fill_in 'Password', with: 'passwd'
+    click_button 'Login'
+    visit '/account/channels'
+
+    click_link 'Add New Channel'
+    fill_in 'Name', with: CHANNEL_NAME
+    select = page.find('.js-select2-contents')
+    select.find("option[value='#{video.id}']").select_option
+    select.find("option[value='#{gallery.id}']").select_option
```

Comment:
note: here u dont use select 2 selectors

Reply:
OK for now (Igor)